packages/run-protocol/src/my-lien.js

```diff
@@ -30,9 +30,32 @@ export const makeStakeReporter = (bridgeManager, stake, denom = 'ubld') => {
   const { make: makeAmt } = AmountMath;
   /** @param {string} numeral */
   const toStake = numeral => makeAmt(stake, BigInt(numeral));
+  /**
+   * @param {string} address
+   * @param {bigint} delta
+   * @returns {Promise<Amount<`nat`>>}
+   */
+  const changeLiened = async (address, delta) => {
+    assert.typeof(address, 'string');
+    const newAmount = await E(bridgeManager).toBridge(XLien.name, {
+      type: XLien.LIEN_CHANGE_LIENED,
+      address,
+      denom,
+      delta: `${delta}`,
```

Comment:
what happens if this isn't stringified?

if `toBridge` requires the second argument to be `Record<string, string>` let's annotate that.

Reply:
I *think* the bridge device just JSON-ifies whatever data is passed. I started down the rabbit hole to see where this goes and got lost. In any case, I'm confident that I'm following the established pattern from the old version, and I delegate further questions to the real experts.